index.html

```diff
@@ -27,13 +27,19 @@ <h2>TUD Informatik Bachelor VV {{semester}}</h2>
                 <button class="button-show-hidden-courses" ng-if="hiddenCourses().length > 0" class="border-left" ng-click="config.showHiddenCoursesDialog = true">
                     {{hiddenCourses().length}} ausgeblendete Kurse anzeigen
                 </button>
-                <div data-angular-treeview="true" data-tree-id="tree" data-tree-model="data" data-node-id="label" data-node-label="label" data-node-children="children" data-label-template="((node.label)) (( labelForNode(node) ))<span class='credits no-wrap credits-((node.credits))' ng-if='node.credits'>((node.credits)) CP</span>">
+                <div data-angular-treeview="true" data-tree-id="tree" data-tree-model="data" data-node-id="label" data-node-label="label" data-node-children="children" data-label-template="<span class='annotation-((node.annotation))'>((node.label)) (( labelForNode(node) ))</span><span class='credits no-wrap credits-((node.credits))' ng-if='node.credits'>((node.credits)) CP</span>">
```

Comment:
I know this is more of my fault but can you please break this rather long line into more lines?